### PR TITLE
Move to 9.0.0-preview.6.24327.7

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

This pull request includes updates to various project files to upgrade the .NET SDK and several package dependencies to their latest preview versions.

### SDK and Package Updates:

* [`DotnetRuntimeMetadata.json`](diffhunk://#diff-88093b111af50e8b692b4e3bf43a2b32e348ce8e72d0f3f613f84e7e4aa6f76bL3-R6): Updated the SDK channel to `9.0.1xx-preview6` and the package version pattern to `9.0.0-preview.6`.
* [`global.json`](diffhunk://#diff-8df3cd354bc584349d04ad5675b33c042d8b99b741b8b95af394c55e0f5001bfL3-R3): Updated the SDK version to `9.0.100-preview.6.24328.19`.

### Project File Updates:

* [`src/Microsoft.PowerShell.Commands.Diagnostics/Microsoft.PowerShell.Commands.Diagnostics.csproj`](diffhunk://#diff-60550985519599639503c16cbedba057e1d03f2b345450698ed26ef706e934a8L11-R11): Updated `System.Diagnostics.PerformanceCounter` to version `9.0.0-preview.6.24327.7`.
* [`src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj`](diffhunk://#diff-c685efeb82dd23385566a8241eb3db56068963c14299d2054fc57ceaefb4bdc2L50-R50): Updated `System.ServiceProcess.ServiceController` to version `9.0.0-preview.6.24327.7`.
* [`src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj`](diffhunk://#diff-1d477db337d41a086f43d0d9351448c5781d5818a906fb88b55d5e08cf755509L36-R37): Updated `System.Threading.AccessControl` and `System.Drawing.Common` to version `9.0.0-preview.6.24327.7`.
* [`src/Microsoft.PowerShell.CoreCLR.Eventing/Microsoft.PowerShell.CoreCLR.Eventing.csproj`](diffhunk://#diff-e773c7ec18ac501fffcd58beed6fdd6ec4242edd73a7a337464e013b23c96d9aL11-R11): Updated `System.Diagnostics.EventLog` to version `9.0.0-preview.6.24327.7`.
* [`src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj`](diffhunk://#diff-9b88575be5ea8ea6e9597c67f8c284e5bc75529b70c7c61e9762f02022eb78dfL24-R26): Updated multiple packages including `System.IO.Packaging`, `System.Net.Http.WinHttpHandler`, and `System.Text.Encodings.Web` to version `9.0.0-preview.6.24327.7`.
* [`src/Microsoft.WSMan.Management/Microsoft.WSMan.Management.csproj`](diffhunk://#diff-eeac85d4488b2b1f7b7b5570abb13934a47f493cd2a858bfdf84bc05a67679c7L13-R13): Updated `System.ServiceProcess.ServiceController` to version `9.0.0-preview.6.24327.7`.
* [`src/System.Management.Automation/System.Management.Automation.csproj`](diffhunk://#diff-7f76b862df0ba75765a372c47d13cd6efc2d81f5abe6a6580531d104ef1d6279L37-R46): Updated multiple packages including `Microsoft.Win32.Registry.AccessControl`, `System.Configuration.ConfigurationManager`, and `System.Diagnostics.DiagnosticSource` to version `9.0.0-preview.6.24327.7`.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
  - [ ] None
  - **OR**
  - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.5/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
    - [ ] Experimental feature name(s): 
- **User-facing changes**
  - [ ] Not Applicable
  - **OR**
  - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
  - [ ] N/A or can only be tested interactively
  - **OR**
  - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
  - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
  - **OR**
  - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
    - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
      - [ ] Issue filed: 
    - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
      - [ ] Issue filed: 
    - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
      - [ ] Issue filed: 
    - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
      - [ ] Issue filed:

### CODE REVIEW

File: DotnetRuntimeMetadata.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "channel": "8.0.1xx",
+    "channel": "9.0.1xx-preview6",
     "quality": "daily",
     "qualityFallback": "preview",
-    "packageVersionPattern": "8.0.0",
+    "packageVersionPattern": "9.0.0-preview.6",
     "sdkImageVersion": "8.0.101",
```
Comment (adityapatwardhan) [inline DotnetRuntimeMetadata.json:7]: This should be updated

File: DotnetRuntimeMetadata.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     "nextChannel": "8.0.1xx",
```
Comment (adityapatwardhan) [inline DotnetRuntimeMetadata.json:8]: Same here

File: test/tools/TestService/TestService.csproj [hunk 3]
```diff
@@ -14,6 +14,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.Compatibility" Version="8.0.0" />
+    <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```
Comment (adityapatwardhan) [inline test/tools/TestService/TestService.csproj:17]: should be `9.0.0-preview.6.24327.7`

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: test/tools/TestService/TestService.csproj (hunk 3 above)
Comment (TravisEz13) [reply test/tools/TestService/TestService.csproj:17]:
```suggestion
    <PackageReference Include="System.Formats.Asn1" Version="9.0.0-preview.6.24327.7" />
```

File: DotnetRuntimeMetadata.json (hunk 2 above)
Comment (TravisEz13) [reply DotnetRuntimeMetadata.json:8]:
```suggestion
    "nextChannel": "9.0.0-rc.1",
```

File: DotnetRuntimeMetadata.json (hunk 1 above)
Comment (TravisEz13) [reply DotnetRuntimeMetadata.json:7]:
```suggestion
    "sdkImageVersion": "9.0.0-preview.6.24327.7",
```

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**